package/src/renderer/HostConfig.ts

```diff
@@ -101,11 +101,11 @@ const removeNode = (parent: Node, child: Node) => {
   bustBranchMemoization(parent);
   const index = parent.children.indexOf(child);
   parent.children.splice(index, 1);
-  child.depMgr.unSubscribeNode(child);
   // unsubscribe to all children as well
   for (const c of child.children) {
-    removeNode(child, c);
```

Comment:
The change here doesn't look correct, can you adjust it? I actually looks like this file should have 0 diffs, could that be?

Reply:
we remove `node.removeNode()` but `removeNode()` stays. I think this equals 0 diffs unless I'm mistaken.
If we take a step back, it makes sense that this file wouldn't have a diff?

Reply:
No, there was a mistake in the reconciler where it didn't remove child nodes - which is why this was added.